lib/src/chip/chip.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'hc-chip',
+    templateUrl: './chip.component.html',
+    styleUrls: ['./chip.component.scss']
+})
+export class ChipComponent implements OnInit {
+
+    @Input() action: boolean = false;
```

Comment:
Same as the ChipRowComponent, you'll want to create a getter and setter to handle the user setting a string value to this input